src/NUnitConsole/nunit3-console.tests/ResultReporterTests.cs

```diff
@@ -101,7 +101,7 @@ public void SummaryReportTest()
             var expected = new string[] {
                 "Test Run Summary",
                 "    Overall result: Failed",
-                "   Tests run: 35, Passed: 30, Errors: 1, Failures: 3, Inconclusive: 1",
+                "   Tests run: 35, Passed: 30, Failed: 3, Errors: 1, Inconclusive: 1",
                 "     Not run: 10, Invalid: 3, Ignored: 4, Explicit: 3, Skipped: 0",
```

Comment:
Let's discuss what we want here. Using the internal terminology, there are seven tests with TestStatus.Failed:
 * 3 Failures "Failed" (ResultState.Failure)
 * 1 Error      "Failed:Error" (ResultState.Error)
 * 3 Invalid    "Failed:Invalid" (ResultState.NotRunnable)
That is, we currently have three different ways a test may be "Failed."

FWIW, we also have three ways a test may be skipped: "Skipped:Ignored", "Skipped:Explicit" and just plain "Skipped".

It's clear that our external description of this doesn't match the internal implementation perfectly, but maybe that's not a real problem. In a future update, maybe we should change this to first report the four basic status counts and then add other lines to break down those things that can be broken down. That is...

```
Tests run: 35, Passed: 30, Failed: 7, Inconclusive: 1, Skipped: 7
    Failed Tests - Failures: 3, Errors: 1, Invalid: 3
    Skipped Tests - Ignroed: 4, Explicit: 3, Other: 0
```

As I said, let's discuss before going further. Do we want to make our internals match user perception? Or do we want to push the users to understanding our internals?